src/airtable/project_summary.js

```diff
@@ -169,11 +169,13 @@ const summarize = (proposals) => {
     if (!project) {
       project = {
         'Project Name': proposal['Project Name'],
-        'Voted Yes': proposal['Voted Yes'] ?? 0,
-        'Voted No': proposal['Voted No'] ?? 0,
-        'OCEAN Granted': proposal['OCEAN Granted'] ?? 0,
-        'Times Proposed': 1,
-        'Times Granted': proposal['OCEAN Granted'] > 0 ? 1 : 0,
+        'Voted Yes': 0,
```

Comment:
Why do we always set this to 0? If there is an older proposal shouldn't it be proposal['Voted Yes']?

Reply:
After line 190 I update the values, this is the initial 'project' object.

Reply:
But can't we set it directly in here so we have less code lines?

Reply:
It used to work like that previously: https://github.com/oceanprotocol/DAOBot/blob/main/src/airtable/project_summary.js#L169

I think this approach is much cleaner as we are setting the values in one place.

before:
if the project doesn't exist
initialize the project object and set the values for each field
else
update each field

after:
if the project doesn't exist
initialize an empty project object
update/set each field